cookbooks/bcpc-hadoop/recipes/phoenixqs_hbase_site.rb

```diff
@@ -0,0 +1,14 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2
+
+include_recipe 'bcpc-hadoop::phoenixqs_kerberos'
+
+site_xml = node.default[:bcpc][:hadoop][:hbase][:site_xml]
+generated_values = {}
```

Comment:
Can we add the property ``phoenix.queryserver.http.port`` so that the default value can be overwritten when required?

Reply:
in the environment yuo can add node['bcpc']['hadoop']['hbase']['site_xml'] as an override without having it declared explicitly